validation-test/stdlib/CollectionType.swift.gyb

```diff
@@ -8,6 +8,9 @@
 // <rdar://problem/24357067> The compiler never finishes compiling validation-test/stdlib/CollectionType.swift.gyb in -O
 // REQUIRES: swift_test_mode_optimize_none
 
+// FIXME: This test takes too long to complete on Android.
+// UNSUPPORTED: OS=linux-androideabi
```

Comment:
Is there a way we could split it up?  Maybe something like `validation-test/stdlib/Slice`?